### PR TITLE
Fix test target excludes

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- String comparision was broken because target
  in the loop was a GStringImpl which is different
  than the excludes list which are Strings.
- Also simplify extraTestLabels local def since
  we've already guaranteed not null when it was
  defined earlier.

related #8421
[skip ci]
Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Jenkins test sanity xlinuxcm

**Comment by AdamBrousseau** [comment]
Jenkins test sanity.functional xlinuxcm jdk11

**Comment by AdamBrousseau** [comment]
Jenkins test sanity xlinuxcm jdk11

**Comment by AdamBrousseau** [comment]
@vsebe for review
sanity.functional
https://ci.eclipse.org/openj9/job/Pipeline_Build_Test_JDK11_x86-64_linux_cm/877/
sanity
https://ci.eclipse.org/openj9/job/Pipeline_Build_Test_JDK11_x86-64_linux_cm/878/

File: buildenv/jenkins/common/pipeline-functions.groovy [hunk 1]
```diff
@@ -435,11 +435,9 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
             }
             echo "TEST_FLAG:'${TEST_FLAG}'"
 
-            def extraTestLabels = EXTRA_TEST_LABELS[target] ?: ''
-            echo "Target:'${target}' extraTestLabels:'${extraTestLabels}'"
-
+            def extraTestLabels = EXTRA_TEST_LABELS[target]
```
Comment (vsebe) [inline buildenv/jenkins/common/pipeline-functions.groovy]: This change is unrelated to this PR.

Comment (AdamBrousseau) [reply buildenv/jenkins/common/pipeline-functions.groovy]: Updated
